package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "bienestar-animal",
-  "version": "0.0.0",
+  "version": "1.0.0",
```

Esto no recuerdo como estaba... estaba como 1.0.0?
```suggestion
  "version": "0.0.0",
```